libs/ui-shared/src/lib/components/atoms/toaster/Toaster.js

```diff
@@ -0,0 +1,3 @@
+import { ToastContainer, toast } from "react-toastify";
```

Comment:
add react-toastify, where can we use it?

Reply:
I removed toast component for now, We are not using it currently.